tox.ini

```diff
@@ -37,6 +37,7 @@ commands =
     lib: pip install --editable .[dev]
     lib: make flake8
     lib: make pylint
+    lib: make imports
```

Comment:
maybe check timings but this is probably fastest (or second after flake8) so may be worth running before the slower pylint